Tests/IntegrationTests/IntegrationTests.Shared/Fixtures/IntegrationTestFixture.cs

```diff
@@ -1,82 +1,6 @@
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.TestHost;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Logging;
-using Xunit;
-using Xunit.Abstractions;
-
 namespace IntegrationTests.Shared.Fixtures;
 
-public class IntegrationTestFixture : IAsyncLifetime
+public class IntegrationTestFixture
 {
-    private readonly CustomWebApplicationFactory<Program> _factory;
-
-    public IntegrationTestFixture()
-    {
-        //hack for being able to run integration tests in azuer/github containers
-        Environment.SetEnvironmentVariable("DOTNET_hostBuilder:reloadConfigOnChange", "false");
```

Comment:
@mehdihadeli you will have to set this environment variable before any testhosts/factories are created. without this, the pipelines in github/azure are crapping out. i don't really know exactly why it's happenning. maybe a because a testhost is being created per test execution? i'm not too familiar with xunit, is there a hook/place where this env-var can be set on xunit initialization/startup?

Reply:
I put your mentioned config in the correct place.